src/DssSpell.sol

```diff
@@ -59,187 +51,224 @@ contract DssSpellAction is DssAction {
 
-            // Init new OracleWrapper with clKey: "FLAP_SKY_ORACLE"
-            "FLAP_SKY_ORACLE"
-        );
+        // Stability Fee (SF) changes:
+        // Note: only heading, changes follow
 
-        // Additional Actions
+        // WBTC-A: Increase by 1.5 percentage points, from 7.75% to 9.25%
+        DssExecLib.setIlkStabilityFee("WBTC-A", NINE_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        // Increase vow.hop by 1386 seconds from 10249 seconds to 11635 seconds
-        DssExecLib.setValue(MCD_SPLIT, "hop", 11_635);
+        // WBTC-B: Increase by 1.5 percentage points, from 8.25% to 9.75%
+        DssExecLib.setIlkStabilityFee("WBTC-B", NINE_PT_SEVEN_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        // Decrease vow.bump by 40000 USDS from 65000 USDS to 25000 USDS
-        DssExecLib.setValue(MCD_VOW, "bump", 25_000 * RAD);
+        // WBTC-C: Increase by 1.5 percentage points, from 7.5% to 9%
+        DssExecLib.setIlkStabilityFee("WBTC-C", NINE_PCT_RATE, /* doDrip = */ true);
 
-        // Note: bump minor chainlog version due to the new flapper contract
-        DssExecLib.setChangelogVersion("1.19.0");
+        // Dai & SKY Savings Rate:
+        // Note: only heading, changes follow
 
-        // ---------- Sky Ecosystem Liquidity Bootstrapping ----------
-        // Forum: https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal-week-of-2024-09-23/25179
-        // Poll: https://vote.makerdao.com/polling/QmfYzKTh
+        // DSR: Decrease by 0.5 percentage points, from 6% to 5.5%
+        DssExecLib.setDSR(FIVE_PT_FIVE_PERCENT_RATE, /* doDrip = */ true);
 
-        // Transfer 10,000,000 DAI to the Pause Proxy from the Surplus Buffer
-        DssExecLib.sendPaymentFromSurplusBuffer(MCD_PAUSE_PROXY, 10_000_000);
+        // SSR: Increase by 0.25 percentage points, from 6.25% to 6.5%
+        SUsdsLike(SUSDS).drip();
+        SUsdsLike(SUSDS).file("ssr", SIX_PT_FIVE_PERCENT_RATE);
 
-        // Note: we have to approve DAI_USDS contract to convert DAI into USDS
-        DAI.approve(DAI_USDS, 10_000_000 * WAD);
+        // ---------- Update PSM-USDC-A Fees  ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2
+        // Poll: https://vote.makerdao.com/polling/QmRjrFYG
 
-        // Convert 10,000,000 DAI to USDS using DAI_USDS
-        // Note: this is done by the next line of code
+        // PSM-USDC-A tin: Decrease by 0.01 percentage points, from 0.01% to 0%
+        // Note: this is done via the DssLitePsmMigrationPhase3 script: line 78
 
-        // Transfer 10,000,000 USDS from PauseProxy to 0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD
-        DaiUsdsLike(DAI_USDS).daiToUsds(LIQUIDITY_BOOTSTRAPPING, 10_000_000 * WAD);
+        // PSM-USDC-A tout: Decrease by 0.01 percentage points, from 0.01% to 0%
+        // Note: this is done via the DssLitePsmMigrationPhase3 script: line 79
```

Comment:
Some of the instructions that are done in script shows up under the section title and others shows up below all the lite-p3m section titles. Could we unify this to one shape?

Reply:
this is done because that is the order in the excel sheet. Additionally, `Update PSM-USDC-A Fees` is a different set of instructions that is executed implcitly in the migration script. How do you recommend to do it?

Reply:
With the reason you specified, I'm okay keeping it as it is.